src/vm/runners/cairo_runner.rs

```diff
@@ -48,10 +48,21 @@ pub struct CairoRunner {
 }
 
 impl CairoRunner {
-    pub fn new(program: &Program) -> Result<CairoRunner, RunnerError> {
+    pub fn new(program: &Program, layout: Option<String>) -> Result<CairoRunner, RunnerError> {
+        let layout = layout.unwrap_or_else(|| String::from("plain"));
```

Comment:
I think given it makes sense to just handle this at the option level with a default value and drop the `Option<_>`.

Reply:
Changed (7735799)